### ISSUE TITLE
Linking .asperite file to .blend file not working if they are stored on different drives

### ISSUE THREAD

**Issue by powertomato** [lampysprites/pribambase#34]
**Reality**
It's a Windows issue. If the .asperite and .blend file are stored on different drives (e.g. C: and D:) "Add to .blend file" from the Asperite-Plugin fails on Blender-Side with the following Error: `ValueError: path is on mount 'D:', start on mount 'C:'`

**Expectation**
I expect it to work regardless of the location of the files.

**Logs**

```py
Python: Traceback (most recent call last):
  File "C:\Users\power\AppData\Roaming\Blender Foundation\Blender\3.3\scripts\addons\pribambase\util.py", line 101, in modal
    self.modal_execute(context)
  File "C:\Users\power\AppData\Roaming\Blender Foundation\Blender\3.3\scripts\addons\pribambase\modify.py", line 538, in modal_execute
    bpy.ops.pribambase.sprite_open(filepath=self.path, relative=addon.prefs.use_relative_path, sheet=self.sheet, layers=self.layers)
  File "C:\Program Files\Blender Foundation\Blender 3.3\3.3\scripts\modules\bpy\ops.py", line 113, in __call__
    ret = _op_call(self.idname_py(), None, kw)
RuntimeError: Error: Python: Traceback (most recent call last):
  File "C:\Users\power\AppData\Roaming\Blender Foundation\Blender\3.3\scripts\addons\pribambase\image.py", line 258, in execute
    bpy.ops.pribambase.sprite_stub(name=name, source=self.filepath, layers=self.layers, sheet=self.sheet)
  File "C:\Program Files\Blender Foundation\Blender 3.3\3.3\scripts\modules\bpy\ops.py", line 113, in __call__
    ret = _op_call(self.idname_py(), None, kw)
RuntimeError: Error: Python: Traceback (most recent call last):
  File "C:\Users\power\AppData\Roaming\Blender Foundation\Blender\3.3\scripts\addons\pribambase\image.py", line 334, in execute
    img.sb_props.source_set(self.source)
  File "C:\Users\power\AppData\Roaming\Blender Foundation\Blender\3.3\scripts\addons\pribambase\props.py", line 253, in source_set
    self.source = bpy.path.relpath(source)
  File "C:\Program Files\Blender Foundation\Blender 3.3\3.3\scripts\modules\bpy\path.py", line 100, in relpath
    return "//" + _os.path.relpath(path, start)
  File "C:\Program Files\Blender Foundation\Blender 3.3\3.3\python\lib\ntpath.py", line 718, in relpath
    raise ValueError("path is on mount %r, start on mount %r" % (
ValueError: path is on mount 'D:', start on mount 'C:'
Location: C:\Program Files\Blender Foundation\Blender 3.3\3.3\scripts\modules\bpy\ops.py:113
Location: C:\Program Files\Blender Foundation\Blender 3.3\3.3\scripts\modules\bpy\ops.py:113
```

**Version info**
- for Blender: 3.3
- for Aseprite: v1.3-rc2 (dev-version)
- for Pribambase: v2.5.1
- OS: Win 11

**Comment by lampysprites** [comment]
Huh I see. Meanwhile, as a workaround, it probably works in sprite menu in
uv editor, after unchecking "relative" option on the right of the file
dialog.

**Comment by lampysprites** [comment]
fixed in 45c6ebd
